osgp/platform/osgp-adapter-domain-core/src/test/java/org/opensmartgridplatform/adapter/domain/core/application/services/ConfigurationManagementServiceTest.java

```diff
@@ -0,0 +1,275 @@
+    @Mock
+    private OsgpCoreRequestMessageSender osgpCoreRequestMessageSender;
+    @Mock
+    private Ssld ssld;
+    @Mock
+    private Device device;
+    @Captor
+    private ArgumentCaptor<RequestMessage> requestMessageArgumentCaptor;
+    @Captor
+    private ArgumentCaptor<String> messageTypeArgumentCaptor;
+    @Captor
+    private ArgumentCaptor<String> ipAddressArgumentCaptor;
+    @Captor
+    private ArgumentCaptor<Integer> messagePriorityArgumentCaptor;
+    @Captor
+    private ArgumentCaptor<Long> scheduledTimeArgumentCaptor;
+    @Captor
+    private ArgumentCaptor<ResponseMessage> responseMessageArgumentCaptor;
+    @InjectMocks
+    @Qualifier("organisationDomainService")
+    private ConfigurationManagementService configurationManagementService;
+
+    private static final String ORG_IDENTIFICATION = "orgIdentification";
+    private static final String DEVICE_IDENTIFICATION = "deviceIdentification";
+    private static final String CORRELATION_UID = "correlationUid";
+    private static final long SCHEDULE_TIME = 1;
+    private static final String MESSAGE_TYPE = "none";
+    private static final int MESSAGE_PRIORITY = 1;
+    private static final String IP_ADDRESS = "333.333.1.22";
+
+    private OsgpException exception;
+    private static final ByteArrayOutputStream OUT_CONTENT = new ByteArrayOutputStream();
+    private static final Organisation TEST_ORGANISATION = new Organisation();
+
+    @BeforeEach
+    public void setUp() throws NoSuchFieldException,
+            SecurityException, IllegalArgumentException, IllegalAccessException {
+
+        // make correlationIds
+        this.correlationIds = new CorrelationIds(ORG_IDENTIFICATION, DEVICE_IDENTIFICATION, CORRELATION_UID);
+        this.configurationManagementService = new ConfigurationManagementService();
+
+        // do injection using reflection
+        // Inject Mocks doesn't inject these mocks, possibly because they are in the parent class of
+        // ConfigurationManagementService
+        this.injectionUsingReflection("organisationDomainService",
+                this.configurationManagementService, this.organisationDomainService);
+        this.injectionUsingReflection("deviceDomainService", this.configurationManagementService,
+                this.deviceDomainService);
+        this.injectionUsingReflection("domainCoreMapper", this.configurationManagementService,
+                this.domainCoreMapper);
+        this.injectionUsingReflection("osgpCoreRequestMessageSender",
+                this.configurationManagementService, this.osgpCoreRequestMessageSender);
+        this.injectionUsingReflection("webServiceResponseMessageSender",
+                this.configurationManagementService, this.webServiceResponseMessageSender);
+        this.injectionUsingReflection("ssldRepository", this.configurationManagementService,
+                this.ssldRepository);
+
+        System.setOut(new PrintStream(this.OUT_CONTENT));
```

Comment:
The static field ConfigurationManagementServiceTest.OUT_CONTENT should be accessed in a static way.
So replace this.OUT_CONTENT with OUT_CONTENT
goes for all the occurences of this.OUT_CONTENT